src/Serialization/upgrades/0.11.2.jl

```diff
@@ -0,0 +1,91 @@
+################################################################################
+# Upgrade Summary
+# Types of the form `Vector{T}` are only serialized with backref when
+# `is_basic_serialization_type(T) == false`. If
+# `is_basic_serialization_type(T) == true` the type `Vector{T}` does not
+# serialize `T` in the entries, but instead as a part of the `Vector` serialization.
+# Instances of the form `Tuple{T}` with `is_basic_serialization_type(T) == true` 
+# are no longer serialized with backref, however these types were already storing
+# there types outside of the JSON array of entries
+
+push!(upgrade_scripts, UpgradeScript(
+    v"0.11.2",
+    function (s::DeserializerState, dict::Dict) 
+        if haskey(dict, :_ns)
+            haskey(dict[:_ns], :polymake) && return dict
+        end
+    
+        if !haskey(dict, :type)
+            upgraded_dict = Dict{Symbol, Any}()
+            for (key, value) in dict
+                if value isa String
+                    upgraded_dict[key] = value
+                else
+                    # recursive call
+                    upgraded_dict[key] = var"#self#"(s, value)
```

Comment:
What is this about? Where is this `var"#self#"` coming from?

Reply:
so var"#self" is how you can recursively call an unnamed function, seems to currently be the only way to do this in julia, I'll add that to comment